src/scripts/main.js

```diff
@@ -1,5 +1,42 @@
 'use strict';
 
+const spider = document.querySelector('.spider');
+const wall = document.querySelector('.wall');
+
 document.addEventListener('click', e => {
   // write code here
+
+  // console.log(e.target.className);
+
+  if (e.target.className === 'wall') {
+    const spiderWidth = parseFloat(
+      window.getComputedStyle(spider).width);
+    const spiderHeight = parseFloat(
+      window.getComputedStyle(spider).height);
+
+    const wallWidth = parseFloat(
+      window.getComputedStyle(wall).width);
+    const wallHeight = parseFloat(
+      window.getComputedStyle(wall).height);
```

Comment:
you can create some functions like 
```
const getWidth = (element) => parseFloat(
  window.getComputedStyle(spider).width
)
```
(create for height as well)

and use like


```
const spiderWidth = getWidth(spider)
const wallWidth = getWidth(wall)
```